packages/core/test/utils/matchers.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace jest {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    interface Matchers<R> {
+      toBeSortedEqual: (expected: Iterable<string>) => CustomMatcherResult
+    }
+  }
+}
+
+export function installMatchers (): void {
+  expect.extend({
+    toBeSortedEqual (this: jest.MatcherContext, received: Iterable<string>, expected: Iterable<string>): jest.CustomMatcherResult {
```

Comment:
could this be
```suggestion
    toBeSortedEqual (this: jest.MatcherContext, received: Iterable<any>, expected: Iterable<any>): jest.CustomMatcherResult {
```

to be of more general use for the *future*™?

Reply:
It was causing type issues™ and I'd like to let myself of the future solve it :D